api/dimensions_test.go

```diff
@@ -15,8 +15,40 @@ import (
 	. "github.com/smartystreets/goconvey/convey"
 )
 
+// func to unmarshal and validate body bytes
+var validateBody = func(bytes []byte, expected models.DatasetDimensionResults) {
+	var response models.DatasetDimensionResults
+	err := json.Unmarshal(bytes, &response)
+	So(err, ShouldBeNil)
+	So(response, ShouldResemble, expected)
+}
+
 func TestGetDimensionsReturnsOk(t *testing.T) {
 	t.Parallel()
+
+	mongoData := []bson.M{
+		bson.M{"doc": bson.M{}},
```

Comment:
It isn't necessary to explicitly set the type to bson.M on lines 30 and 31 as this is inferred by the initialisation of the array of bson.M objects on line 29